### PR TITLE
Make sure credentials form and respective network calls only happen if Rewind is active

### PR DESCRIPTION
**Pull request by rcoll**
Currently, the credentials form will show for all Jetpack sites. This is incorrect behavior for a number of reasons, but currently it's important that only Pressable users see the form. In order to rectify this, while also making the form available for testing on non-Pressable sites, we should wall off the form and associated network requests for credentials to only sites where `isRewindActive` is true.

**Testing instructions:**
- Make sure the form at Settings > Security is visible for a Pressable site.
- Make sure the form at Settings > Security is visible for a non-Pressable site that has Rewind active.
- Make sure the form is hidden and no network requests are made to `/get-credentials` on a site without Rewind active.

### CODE REVIEW

**Comment by eliorivero** [comment]
Works as intended, 🐑 !

File: client/my-sites/site-settings/settings-security/main.jsx [hunk 1]
```diff
@@ -62,18 +64,20 @@ const SiteSettingsSecurity = ( { site, siteId, siteIsJetpack, translate } ) => {
 
 	return (
 		<Main className="settings-security__main site-settings">
+			<QueryRewindStatus siteId={ siteId } />
```
Comment (eliorivero) [inline client/my-sites/site-settings/settings-security/main.jsx]: Shouldn't you use this method?
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/activity-log/index.jsx#L598

Comment (rcoll) [reply client/my-sites/site-settings/settings-security/main.jsx]: Won't `<QueryRewindStatus />` return true always? Even if the network comes back and says rewind is *not* active?

Comment (rcoll) [reply client/my-sites/site-settings/settings-security/main.jsx]: Perhaps that might be better. I'll try it.

Comment (rcoll) [reply client/my-sites/site-settings/settings-security/main.jsx]: Fixed :-) Thanks.

**Pull request merged**